charts/ingress-nginx/templates/admission-webhooks/cert-manager.yaml

```diff
@@ -47,6 +50,9 @@ metadata:
 spec:
   secretName: {{ include "ingress-nginx.admissionWebhooks.fullname" . }}
   duration: {{ .Values.controller.admissionWebhooks.certManager.admissionCert.duration | default "8760h0m0s" | quote }}
+  {{- with .Values.controller.admissionWebhooks.certManager.admissionCert.revisionHistoryLimit }}
+  revisionHistoryLimit: {{ . }}
+  {{- end }}
```

Comment:
Please do not use `with` for simple values. This changes the context and can lead to unexpected side effects. I'm personally totally fine with `with` for objects, but it's not adding a lot of value for simple types.